docs/concepts/policy/pod-security-policy.md

```diff
@@ -1,97 +1,371 @@
+
+```
+$ kubectl-user create -f- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      pause
+spec:
+  containers:
+    - name:  pause
+      image: gcr.io/google-containers/pause
+EOF
+Error from server (Forbidden): error when creating "STDIN": pods "pause" is forbidden: no providers available to validate pod request
+```
+
+**What happened?** Although the PodSecurityPolicy was created, neither the
+created pod nor `fake-user` have permission to use the new policy:
+
+```shell
+$ kubectl-user auth can-i use podsecuritypolicy/example.unprivileged
+no
+```
+
+Create the rolebinding to grant `fake-user` the `use` verb on the example
+policy:
+
+_Note: This is not the recommended way! See the [next section](#run-another-pod)
+for the preferred approach._
+
+```shell
+$ kubectl-admin create role psp:unprivileged \
+    --verb=use \
+    --resource=podsecuritypolicy \
+    --resource-name=example.unprivileged
+role "psp:unprivileged" created
+$ kubectl-admin create rolebinding fake-user:psp:unprivileged \
+    --role=psp:unprivileged \
+    --serviceaccount=psp-example:fake-user
+rolebinding "fake-user:psp:unprivileged" created
+$ kubectl-user auth can-i use podsecuritypolicy/example.unprivileged
+yes
+```
+
+Now retry creating the pod:
+
+```shell
+$ kubectl-user create -f- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      pause
+spec:
+  containers:
+    - name:  pause
+      image: gcr.io/google-containers/pause
+EOF
+pod "pause" created
+```
+
+It works as expected! But any attempts to create a privileged pod should still
+be denied:
+
+```shell
+$ kubectl-user create -f- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      privileged
+spec:
+  containers:
+    - name:  pause
+      image: gcr.io/google-containers/pause
+      securityContext:
+        privileged: true
+EOF
+Error from server (Forbidden): error when creating "STDIN": pods "privileged" is forbidden: unable to validate against any pod security policy: [spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
+```
+
+Delete the pod before moving on:
+
+```shell
+$ kubectl-user delete pause
+```
+
+### Run another pod
+
+Let's try that again, slightly differently:
+
+```shell
+$ kubectl-user run pause --image=gcr.io/google-containers/pause
+deployment "pause" created
+$ kubectl-user get pods
+No resources found.
+$ kubectl-user get events | head -n 2
+LASTSEEN   FIRSTSEEN   COUNT     NAME              KIND         SUBOBJECT                TYPE      REASON                  SOURCE                                  MESSAGE
+1m         2m          15        pause-7774d79b5   ReplicaSet                            Warning   FailedCreate            replicaset-controller                   Error creating: pods "pause-7774d79b5-" is forbidden: no providers available to validate pod request
+```
+
+**What happened?** We already bound the psp:unprivileged role for our fake-user,
```

Comment:
Maybe we need to stand out `psp:unprivileged` and `fake-user` here.

Reply:
I'm not sure what you mean by "stand out". Did you mean add the inline-code backticks?

Reply:
Yes, I did.